assets/app.js

```diff
@@ -984,101 +1345,412 @@ new Vue({
                 this.products[index].descriptions[language] = my_description
             }
         },
+        bulkChangeTranslationStatus: function (type) {
+            proceed = confirm("Are you sure that you want to set all " + this.resourceDict[type]['name'] + " to be fully translated?")
+
+            if (proceed && hasApi) {
+                let resource_translation_requested = this[this.resourceDict[type].translationRequested]
+                let id_list = resource_translation_requested.map(function (a) {
+                    return a.id;
+                })
+
+                let group_resources = [], chunk_size = 900;
+                for (let i = 0; i < id_list.length; i += chunk_size) {
+                    group_resources.push(id_list.slice(i, i + chunk_size));
+                }
+                this.showLoading = true
+
+                let promise_status_list = group_resources.map((item) => {
+                    return new Promise((resolve, reject) => {
+                        fetch(api_endpoint, {
+                            credentials: 'include',
+                            method: 'POST',
+                            body: JSON.stringify({
+                                action: 'bulk_translation_complete',
+                                resources: item,
+                                type: type
+                            })
+                        }).then(function (response) {
+                            return response.json()
+                        }).then(function (response) {
+                            resource_translation_requested.forEach(function (resource) {
+
+                                //verify if resource.id exist in response then switch to translation_requested = 0
+                                if ((response.resources).indexOf(resource.id) > -1) {
+                                    resource.translation_requested = 0
+                                }
+                            })
+                            resolve()
+                        }).catch(function () {
+                            reject()
+                        })
+                    })
+                })
+
+                Promise.all(promise_status_list)
+                    .then(() => {
+                        this.showLoading = false
+                        this.addMessage("Hey, you just updated translation status of all the " + this.resourceDict[type]['name'] + " successfully.", 'success')
+                    })
+                    .catch(() => {
+                        this.showLoading = false
+                        this.addMessage("Sorry, something went wrong!", 'danger')
+                    });
+            }
+        },
+        pushProducts: function(products){
```

Comment:
In ` bulkChangeTranslationStatus()` you use everywhere `let <name>`, in `pushProducts` `var <name>` for the same purpose. Please just use one syntax and change all the others.